src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -8356,31 +8391,32 @@ static void Main()
         public void SynthesizedDelegateTypes_11()
         {
             var source =
-@"class Program
+@"using System;
+class Program
 {
     unsafe static void Main()
     {
         var d1 = int* () => (int*)42;
         var d2 = (int* p) => { };
         var d3 = delegate*<void> () => default;
         var d4 = (delegate*<void> d) => { };
+        Report(d1);
+        Report(d2);
+        Report(d3);
+        Report(d4);
     }
+    static void Report(Delegate d) => Console.WriteLine(d.GetType());
 }";
 
             var comp = CreateCompilation(source, options: TestOptions.UnsafeReleaseExe);
-            comp.VerifyDiagnostics(
-                // (5,18): error CS8917: The delegate type could not be inferred.
-                //         var d1 = int* () => (int*)42;
-                Diagnostic(ErrorCode.ERR_CannotInferDelegateType, "int* () => (int*)42").WithLocation(5, 18),
-                // (6,18): error CS8917: The delegate type could not be inferred.
-                //         var d2 = (int* p) => { };
-                Diagnostic(ErrorCode.ERR_CannotInferDelegateType, "(int* p) => { }").WithLocation(6, 18),
-                // (7,18): error CS8917: The delegate type could not be inferred.
-                //         var d3 = delegate*<void> () => default;
-                Diagnostic(ErrorCode.ERR_CannotInferDelegateType, "delegate*<void> () => default").WithLocation(7, 18),
-                // (8,18): error CS8917: The delegate type could not be inferred.
-                //         var d4 = (delegate*<void> d) => { };
-                Diagnostic(ErrorCode.ERR_CannotInferDelegateType, "(delegate*<void> d) => { }").WithLocation(8, 18));
+            comp.VerifyDiagnostics();
+
+            CompileAndVerify(comp, verify: Verification.Skipped, expectedOutput:
```

Comment:
>expectedOutput [](http://example.com/codeflow?start=65&length=14)

Execute delegates?

Reply:
I guess this is applicable to other tests as well.

Reply:
Same comment as for the other similar thread. I'd expect to see an output difference after the change.